system/react/src/components/Chip.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+
+export const classySelector = '.chip';
+
+export const Chip = styled.button<{
+  role?: 'tab';
+  'aria-selected'?: boolean;
+  'aria-controls'?: string;
+  'data-size'?: 'small' | 'regular';
+}>`
+  --padding-y: 12px;
+  --padding-x: 16px;
+  --padding-small-y: 6px;
+  --padding-small-x: 14px;
+  padding: calc(var(--padding-y) - 2px) calc(var(--padding-x) - 2px);
+  position: relative;
+  text-decoration: none !important;
+  color: var(--text);
+  border-radius: var(--border-radius-full);
+  border: 2px solid var(--border-transparent);
+
+  &:hover {
+    background-color: var(--surface-hover);
+    cursor: pointer;
+  }
+  &:active {
+    background-color: var(--surface-active);
+  }
+  &[data-size='small'] {
+    padding: calc(var(--padding-small-y) - 2px)
+      calc(var(--padding-small-x) - 2px);
```

Comment:
For this we should be able to go with just 2 variables and change the variables to get the change.
```suggestion
    --padding-x: 14px;
    --padding-y: 6px;
```